tests/python/utils.py

```diff
@@ -240,6 +240,37 @@ def check_captured_python_definition(reference_outputs, fd, inputs, device=None)
         raise err
 
 
+# Run original FusionDefinition
+# Clone FusionDefinition
+# Run cloned python definition
+# Check that the result of cloned python definition matches original results
+def check_cpp_translation(reference_outputs, fd, inputs, device=None):
+    try:
+        torch.manual_seed(0)
+        cloned_fd = fd.clone()
+        cloned_outputs = cloned_fd.execute(inputs, device=device)
```

Comment:
Maybe we should print the fusion IR also and compare the output?

Reply:
I was only after functional equivalence for now. Perfect string matching can be tedious.

Reply:
The python and cpp APIs are not a perfect 1:1 match. String matching would be useful for the second round of PRs.